pkg/scheduler/eventhandlers.go

```diff
@@ -174,7 +174,7 @@ func (sched *Scheduler) updatePodInSchedulingQueue(oldObj, newObj interface{}) {
 	if oldPod.ResourceVersion == newPod.ResourceVersion {
 		return
 	}
-	if sched.skipPodUpdate(newPod) {
+	if isAssumed, _ := sched.SchedulerCache.IsAssumedPod(newPod); isAssumed {
```

Comment:
never discard errors

Reply:
it seems the new recommendation is to use `utilruntime.HandleError` in this case.